alab_management/scripts/launch_lab.py

```diff
@@ -57,6 +57,16 @@ def launch_device_manager():
     device_manager.run()
 
 
+def launch_resource_manager():
+    """Launch the resource manager."""
+    from alab_management.resource_manager.resource_manager import ResourceManager
```

Comment:
why do you have the imports inside the function?

Reply:
Because we try not to import any other modules in the command line interface. This is useful mostly for the `config.py`. If you don't import it locally, when you try to do `import alab_management` in a folder without `config.py`, where there will still be an error saying that cannot find the configuration file, which is quite annoying. By putting the import inside the function, it will only be imported when the package truly needs it.

Reply:
hm, not extremely happy with that I will like to see if we can update all places that this is happening!